configure.ac

```diff
@@ -1336,6 +1336,15 @@ if test x$use_ebpf != xno; then
   )
 fi
 
+build_leveldb=yes
+if test x$build_bitcoin_cli$build_bitcoin_tx$build_bitcoin_util$build_bitcoind$bitcoin_enable_qt$use_bench$use_tests = xnonononononono; then
```

Comment:
I guess we should only be checking bitcoind/bitcoin-qt here (the other tools don't need them either?), but perhaps that's safer left for a later PR...

Reply:
You read my mind :)

Reply:
Followup in #24051